app/components/election-time-input.js

```diff
@@ -0,0 +1,58 @@
+// https://github.com/EnCiv/undebate-ssp/issues/6
+
+import { useEffect, useRef, useState } from "react"
+import { createUseStyles } from "react-jss"
+import svg from "../../assets/svg/clock.svg"
+
+const ElectionTimeInput = props => {
+    const { defaultValue = "", onDone = () => {} } = props
+    const [time, setTime] = useState(defaultValue)
+    const classes = useStyles(time)
+    const inputRef = useRef(null)
+
+    useEffect(() => {
+        handleDone(time)
+    }, [])
+
+    const handleChange = e => {
+        setTime(e.target.value)
+    }
+
+    const handleDone = e => {
+        onDone({ valid: time ? true : false, value: time })
+    }
+
+    return (
+        <>
```

Comment:
Not a big deal - but if <> is not needed I'd take it out.  It's amazing you were able to do this with one element.  But it may not remain true.

Reply:
Good catch, forgot to remove them.